src/gsi/read_prepbufr.f90

```diff
@@ -2577,6 +2579,7 @@ subroutine read_prepbufr(nread,ndata,nodata,infile,obstype,lunout,twindin,sis,&
                    if ((kx==188).or.(kx==288) .or.(kx==195) .or.(kx==295)) then
 !                     gustoe=2.5
                       gustoe=1.0
+                      if ( l_rtma3d .and. oerr_gust > 0.0_r_kind ) gustoe = oerr_gust
```

Comment:
The same comment as the above

Reply:
@TingLei-NOAA 
Hi Ting,
Same as my reply to your comment to the line 2560, l_rtma3d is used here to make sure obs error of wind gust is changed only if it is 3DRTMA run, and for most case run, just the default value (1.0m/s) is used. 

Thank you!

-Gang